src/main/webapp/app/entities/quiz-exercise/quiz-exercise-detail.component.ts

```diff
@@ -465,6 +470,28 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, OnDestroy
         return isGenerallyValid && areAllQuestionsValid;
     }
 
+    /**
+     * @function containExplanation
+     * @desc Shows the explanation warning
+     */
+    containExplanation(): boolean {
+        if (this.pendingChanges() && this.validQuiz()) {
+            return true;
+        }
+
+    }
+
+    /**
+     * @function validQuestionTitle
+     * @desc Checks the validity of the question title
+     */
+    warningCheck(): boolean {
```

Comment:
I don't understand this function. This should be one of the invalid reasons. Why is it implement here? Also the name does not fit to the JSDoc comment and you should use braces in the for loop.

Reply:
deleted the fiction and edit it to invalid reasons